### PR TITLE
add module augmentation section to extending-kysely recipe.

### PR DESCRIPTION
**Pull request by igalklebanov**

### CODE REVIEW

File: recipes/extending-kysely.md [hunk 1]
```diff
@@ -340,7 +340,47 @@ db.insertInto('t')
...
-discoverable API.
+You usually don't want to do this. Because of the complex types and typescript's limitations 
+when it comes to inheritence and return types, you'll quickly run into problems.
+Even though Kysely uses classes, Kysely is not designed from the OOP point of view. 
+Classes are used because they are supported natively by Typescript. They provide private 
+variables and a nice discoverable API.
+
+## Extending using module augmentation
+
+You don't want to do this. This method can bypass Typescript's shortcomings when it comes
```
Comment (koskimas) [inline recipes/extending-kysely.md:351]: I think the disclaimer is enough and we can remove the first sentence. If we leave it there, this is a little bit too scary 😄

File: recipes/extending-kysely.md [hunk 2]
```diff
@@ -340,7 +340,47 @@ db.insertInto('t')
...
+Classes are used because they are supported natively by Typescript. They provide private 
+variables and a nice discoverable API.
+
+## Extending using module augmentation
+
+You don't want to do this. This method can bypass Typescript's shortcomings when it comes 
+to inheritence and return types, but relies heavily on Kysely's internal structure, which 
+is destined to change without notice between releases. 
+
+> DISCLAIMER: We do not support this method. Use at your own risk.
```
Comment (koskimas) [inline recipes/extending-kysely.md:355]: 👍

File: recipes/extending-kysely.md (hunk 1 above)
Comment (igalklebanov) [reply recipes/extending-kysely.md:351]: Almost wrote that augmentation is the devil! 👹

**Comment by igalklebanov** [comment]
Thanks for the feedback! 💯 

Remove that scary paragraph.. 😄

**Pull request merged**